cloudhub-2/modules/ROOT/pages/ch2-deploy-api.adoc

```diff
@@ -0,0 +1,180 @@
+     "version": "1.0.5",
+     "packaging": "jar"
+   },
+   "assets": [],
+   "desiredState": "STARTED",
+   "configuration": {
+     "mule.agent.application.properties.service": {
+       "applicationName": "logs-app-uc01",
+       "properties": {},
+       "secureProperties": {}
+     },
+     "mule.agent.logging.service": {
+       "artifactName": "logs-app-uc01",
+       "scopeLoggingConfigurations": []
+     }
+   },
+   "integrations": {
+     "services": {
+       "objectStoreV2": {
+         "enabled": false
+       }
+     }
+   },
+   "vCores": "0.05"
+ }
+}
+----
+
+Deployments that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. If the app is deleted, the ports become available again.
+
+If you configure ports incorrectly, the following errors appear:
+
+* When you specify more than ten ports: `Owner <name>  has reached the maximum port lease capacity`
+* When the port you specify is not in the range of 30500 to 32500: `Ports must be within the supported range of 30500 - 32500`
+* When the port you specify is in use by another app: `Ports 30501 is already in use by other owner`
+
+== Update an App Using the Application Manager API
+
+In the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API], use the `tcp` section under `deploymentSettings` to update TCP port and redeploy your app. 
+
+The following example shows request to update an existing app and its body with a new TCP mapping:
+
+[source,json]
+----
+PATCH
+https://anypoint.mulesoft.com/amc/adam/api/organizations/{organizationId}/environments/{environmentId}/deployments/{deploymentId}
+
+{
+    "id": "id",
+    "target": {
+      "provider": "MC",
+      "targetId": "873a9879-cca7-4211-b90e-826e98123929",
+      "deploymentSettings": {
+        "tcp": {
+          "inbound": {
+            "ports": [
+              {
+                "portNumber": 30507,
+                "applicationPortNumber": 9007
+              },
+              {
+                "portNumber": 30508,
+                "applicationPortNumber": 9008
+              },
+              {
+                "portNumber": 30509,
+                "applicationPortNumber": 9009
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+----
+
+Updates that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. The ports that you replaced with new mappings become available.
+
+== Access Your TCP App
+
+You can access the app using VPN or a transit gateway and within the private space. Since the apps run on worker nodes, they cannot be accessed directly over the internet. The URL to access a TCP app has this structure: `{application-url}.tcp.{environment}.cloudhub.io:{port}`
```

Comment:
Avoid using "since" if you mean because. 
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-synonymsandwordswithmultiplemeanings

Reply:
```suggestion
You can access the app using VPN or a transit gateway and within the private space. Because the apps run on worker nodes, they cannot be accessed directly over the internet. The URL to access a TCP app has this structure: `{application-url}.tcp.{environment}.cloudhub.io:{port}`
```